packages/common/test/projects/HubProject.test.ts

```diff
@@ -118,7 +118,7 @@ describe("HubProject Class:", () => {
     // });
 
     // it("returns editorConfig integrating options", async () => {
-    //   const spy = spyOn(schemasModule, "getEntityEditorSchemas").and.callFake(
+    //   const spy = spyOn(schemasModule, "getEditorSchemas").and.callFake(
```

Comment:
I'd honestly just remove these commented out tests. This one at least doesn't seem relevant anymore